Frameworks/UIKit/UILabel.mm

```diff
@@ -188,10 +188,6 @@ - (void)_initUILabel {
     WXCTextBlock* textBlock = nil;
```

Comment:
nvm. :S

---
In reply to: [108989240](https://github.com/Microsoft/WinObjC/pull/2372#discussion_r108989240) [](ancestors = 108989240,108989011)